### ISSUE TITLE
Authorize decorator not working

### ISSUE THREAD

**Issue by ridyio** [doug-martin/nestjs-query#890]
**Describe the bug**
There seems to be an issue with @Authorize decorator. The filter doesn't applies and is skipped.

### Have you read the [Contributing Guidelines](https://github.com/doug-martin/nestjs-query/blob/master/CONTRIBUTING.md)?
Yes

**To Reproduce**
Create a DTO class & decorate it with Authorize class with a filter. Filter is not applied at all and skipped. I have also created Authorization class and put breakpoints in it's call method but it wasn't called. I also tried putting status value for filter because maybe the context didn't had those but that wasn't applied either. There is no error or anything filter is just skipped. I have rechecked everything many times couldn't find any reason for this from my side. (example below).
```
@ObjectType('Ticket')
@Authorize({
  authorize: (context: UserContext) => ({teamId: {eq: context.req.user.teamId}})
})
export class TicketDTO {
  @FilterableField()
  teamId: number;
}
```

**Expected behavior**
Filter would be applied

**Desktop (please complete the following information):**
 - Node Verson 15.3.0
 - Nestjs-query Version 0.22.0

**Comment by doug-martin** [comment]
@ridyio do you have a custom resolver or are you using the auto generated resolver?

**Comment by ridyio** [comment]
@doug-martin Custom resolver. Here it is:
```
@Resolver(() => TicketDTO)
export class TicketResolver extends CRUDResolver(TicketDTO, {
  CreateDTOClass: CreateTicketInput,
  update: { disabled: true },
  delete: { disabled: true },
  create: { many: {disabled: true } },
  guards: [JwtAuthGuard]
}) {
  constructor(
    ticketService: TicketService
  ) {
    super(ticketService);
  }
}
```

**Comment by doug-martin** [comment]
@ridyio I think this probably the same thing as #864 could you try the solution proposed in there and see if it works for you?

**Comment by ridyio** [comment]
@doug-martin Thanks for solution. Yes i tried it and that seems to do the job.
It could be helpful for some to mention this in documentation so they don't take extending approach like me if authorization is required (Which i think is always). If of course not already mentioned maybe i didn't saw.
Thanks again🙏🏼

**Comment by doug-martin** [comment]
@ridyio this is addressed in the latest version `v0.23.x`. 

I added a new feature to allow specifying a `dtos` option to the `NestjsQueryGraphqlModule` that will not auto generate a resolver if you create a custom one, but still set up all the additional providers for authorizers and hooks. 

https://doug-martin.github.io/nestjs-query/docs/migration-guides/v0.22.x-to-v0.23.x#registering-dtos-when-using-custom-resolvers
